includes/class-coblocks-block-assets.php

```diff
@@ -274,6 +274,24 @@ public function frontend_scripts() {
 			);
 		}
 
+		// Events block.
+		if ( has_block( $this->slug . '/events' ) ) {
+			wp_enqueue_script(
+				$this->slug . '-slick',
```

Comment:
Do we need to enqueue slick here, and within the events/index.php file?

Reply:
Nope, just here is fine.